kstatemachine/src/commonMain/kotlin/ru/nsk/kstatemachine/MetaInfo.kt

```diff
@@ -0,0 +1,11 @@
+package ru.nsk.kstatemachine
+
+sealed interface MetaInfo
+interface StateMetaInfo : MetaInfo
+interface TransitionMetaInfo : MetaInfo
```

Comment:
How do you think maybe StateMetaInfo, TransitionMetaInfo are not necessary? and it is simpler to have single MetaInfo marker interface?

Reply:
Thought about as well, but also thought that you might know better. Will remove

Reply:
Only MetaInfo left